Doc/library/inspect.rst

```diff
@@ -831,6 +832,9 @@ function.
          However, if needed, use :meth:`BoundArguments.apply_defaults` to add
          them.
 
+      .. versionchanged:: 3.8
+         :attr:`arguments` is now of type :class:`dict`.
```

Comment:
To be clearer on what changed consider adding, "Formerly, it was of type OrderedDict"